src/styles/main.scss

```diff
@@ -1,3 +1,70 @@
+@import 'utils/variables';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  width: 710px;
```

Comment:
calendar width should be calculated (depends on cell width and gap between them)